coinmarketcap/src/endpoint/price.ts

```diff
@@ -0,0 +1,61 @@
+import { util } from '@chainlink/ea-bootstrap'
+import { Requester, Validator } from '@chainlink/external-adapter'
+import { ExecuteWithConfig } from '@chainlink/types'
+
+export const NAME = 'price'
+
+const priceParams = {
+  symbol: ['base', 'from', 'coin', 'sym', 'symbol'],
+  convert: ['quote', 'to', 'market', 'convert'],
+  cid: false,
+}
+
+export const execute: ExecuteWithConfig = async (request, config) => {
+  const url = 'https://pro-api.coinmarketcap.com/v1/cryptocurrency/quotes/latest'
```

Comment:
Same comment as above, take `baseUrl` from config obj.

```suggestion
  const url = 'cryptocurrency/quotes/latest'
```